src/Agent.NLog45/Agent.NLog45.csproj

```diff
@@ -37,7 +37,7 @@
       <Version>4.9.0.13</Version>
     </PackageReference>
     <PackageReference Include="NLog">
-      <Version>4.5.0</Version>
+      <Version>4.5.11</Version>
```

Comment:
Is 4.5.11 the minimum version of NLog necessary for this to work?  We generally try to have Loupe target the lowest version for highest compatibility.

Reply:
NLog 4.5.11 is from year 2018, and includes bug-fixes for the JsonSerializer: https://www.nuget.org/packages/NLog/4.5.11